vcd/resource_vcd_nsxv_firewall_rule.go

```diff
@@ -856,14 +859,66 @@ func orgNetworksIdsToNames(networkIds []string, vdc *govcd.Vdc) ([]string, error
 	for index, networkId := range networkIds {
 		orgVdcNetwork, err := vdc.GetOrgVdcNetworkById(networkId, false)
 		if err != nil {
-			return nil, fmt.Errorf("could not find org network with name %s: %s", networkId, err)
+			return nil, fmt.Errorf("could not find org network with ID %s: %s", networkId, err)
 		}
 		orgNetworkNames[index] = orgVdcNetwork.OrgVDCNetwork.Name
 
 	}
 	return orgNetworkNames, nil
 }
 
+// ipSetIdsToNames looks up IP sets by IDs and returns list of their names
+func ipSetIdsToNames(ipSetIds []string, vdc *govcd.Vdc) ([]string, error) {
```

Comment:
Just a question, is there a chance we may need `ipSetIdsToNames` and `ipSetNamesToIds` functions more widely? If yes, please consider moving them to a more accessible file.

Reply:
The other place it will be needed is DHCP relay settings, but that is pretty much the only place. I doubt it deserves a separate file so I moved it into ipset resource's file.